daemon/Cargo.toml

```diff
@@ -34,6 +35,35 @@ bench = false
 doc = false
 required-features = ["executable-deps"]
 
+[features]
+default = ["executable-deps"]
+executable-deps = ["dep:clap", "tokio/signal", "tokio/rt-multi-thread"]
```

Comment:
I moved the features up as they are most often one of the more interesting parts when looking into a `Cargo.toml`. Also, they are more alphabetically sorted with the rest of the keys now too.

Personally I also put the default feature first as it has a special meaning while the others don't. In this case it's also the alphabetical order.